### DISCUSSION TITLE
Clearing the onOffice cache

### DISCUSSION THREAD

**Discussion by fredericalpers** [onOffice-Web-Org/oo-wp-plugin#483] [_Discarded pitches]
### Current state
After clearing the cache there is no success message that the cache was actually cleared.

### Desired state
After clearing the cache there should be a success message. See attached screenshots. (success-message-ger.jpg) and success-message-en.jpg)

E.g.

![success-message-en](https://user-images.githubusercontent.com/88546396/229795847-609d4b6e-3fc5-41b2-b87c-cc8ebb6deccc.jpg)
![success-message-ger](https://user-images.githubusercontent.com/88546396/229795852-97123da3-d517-4bc2-a772-bf16a345c683.jpg)

**Comment by fredericalpers** [comment]
@dai-eastgate Please investigate and let me know how long this will approx. take.

**Reply by dai-eastgate** [reply]
@fredericalpers I need a day to fix and test it.

**Reply by fredericalpers** [reply]
@yeneastgate since all the issues seem to be fixed and its on us to test it one more time, is it possible to squeeze this in this week before the public holidays while resolving the open [dependencies ](https://github.com/onOffice-Web-Org/oo-wp-plugin/pulls?q=is%3Aopen+is%3Apr+label%3Adependencies)?

**Reply by yeneastgate** [reply]
@fredericalpers yes, I will fix it in this week.
